### PR TITLE
35 drinks by bar sad pathedge case

### LINKED ISSUE #35: Drinks by Bar - Sad Path/Edge Case
- [x] Test the return if a field is requested twice in the same query
- [x] Test for a Bar ID requested that is not in DB

### PR DESCRIPTION
**Pull request by RyanChrisSmith**
## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Other (please describe):
       - sad/edge path testing for all drinks for a bar query
       - adjusted rescue_from method in schema to have more accurate and 
       explicit error messages returned

## Problem/feature

_What problem are you trying to solve?_

_Issue Number?_
Fixes #35 

## Checklist

- [x] Code compiles correctly
- [x] Tests for the changes have been added (for bug fixes/features)
- [x] All tests passing
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: spec/graphql/queries/existing_database/get_one_drink_for_a_bar_spec.rb [hunk 1]
```diff
@@ -72,5 +72,118 @@ def query_drink
...
+
+      post '/graphql', params: {query: query_bad_id_drink}
+      result = JSON.parse(response.body)
+
+      expect(result["errors"][0]["message"]).to eq("Couldn't find Drink with 'id'=300")
+    end
+  end
+
+  describe 'edge case' do
+    it 'does not receive and error when a field is requested errently twice' do
```
Comment (aleish-m) [inline spec/graphql/queries/existing_database/get_one_drink_for_a_bar_spec.rb:111]: Check the description for this test I think there is an extra partial word.

**Review by aleish-m — COMMENTED** [review]
Hey @RyanChrisSmith, I left comments about 2 minor things needed on this then I'll merge it in :)

File: spec/graphql/queries/existing_database/get_one_drink_for_a_bar_spec.rb (hunk 1 above)
Comment (RyanChrisSmith) [reply spec/graphql/queries/existing_database/get_one_drink_for_a_bar_spec.rb]: Should be fixed as requested, thanks for noticing!

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "editor.stickyScroll.enabled": false,
```
[the remark this answers is not in the document]
Comment (RyanChrisSmith) [reply .vscode/settings.json:2]: Done and done
